src/main/webapp/js/authentification/authenticationService.js

```diff
@@ -0,0 +1,54 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('ideaFactories')
+        .factory('AuthenticationService', AuthenticationService);
+
+    AuthenticationService.$inject = ['$http', '$cookieStore', '$rootScope', 'usersFactory','md5'];
+    function AuthenticationService($http, $cookieStore, $rootScope, usersFactory, md5) {
+        var service = {};
+
+        service.Login = Login;
+        service.SetCredentials = SetCredentials;
+        service.ClearCredentials = ClearCredentials;
+
+        return service;
+
+        function Login(email, password, callback) {
+
+        var response;
+            usersFactory.getUserByEmailAndPassword(email, password).then(function (user) {
+                response = { success: true };
+                callback(response);}
+            ,function() {
+                response = { success: false, message: 'Email or password is incorrect' };
+                callback(response)
+                });
+
+
+
+        }
+
+        function SetCredentials(email, password) {
+            var authdata = md5.createHash((email + ':' + password) || '');
+
+            $rootScope.globals = {
```

Comment:
May be, it makes sense store on user side only id and required for displaying data (email) (it is seems like password in not required later)?